mlkem/native/x86_64/src/basemul.c

```diff
@@ -8,15 +8,57 @@
 #if defined(MLKEM_NATIVE_ARITH_BACKEND_X86_64_DEFAULT)
 
 #include "consts.h"
+
 #include "poly.h"
 #include "polyvec.h"
 
 #include "arith_native_x86_64.h"
 
-static void poly_basemul_montgomery_avx2(poly *r, const poly *a, const poly *b)
+int16_t zetas_avx2[64] = {
```

Comment:
Can we autogenerate those constants via `autogenerate_files.py`?

Reply:
I tried to do that, but the script doesn't work on my machine (even without any changes to it). I get this error:
```
➜  mlkem-native ./scripts/autogenerate_files.py
Failed to auto-format autogenerated code (clang-format return code 1
```
even though I have `clang-format` installed and in the PATH.

Reply:
@dkostic This is odd. We do test on a host of machines, but your's seems to hit a gap. 

Can you open issues for this and for your nix setup issue please, describing the exact setup and steps you took, and the machine you are working with?

Reply:
sure, I can do that

Reply:
opened #581 and #582

Reply:
Thanks! Looks like version issues, we need to document the requirements better.

For this PR, I suggest you install nix through the official script, which should unblock you.

Reply:
you mean this: https://nixos.org/download/? I did that, but the issue persist.

Reply:
Did you uninstall the apt version and double check `nix --version`? Can you update the issue accordingly?

It would be puzzling if not even the setup script worked, we test this in CI for c7i (see the nix-setup action for what exactly is run).

Reply:
Any update @dkostic?